src/integrationTest/java/com/github/manedev79/timesheet/workingday/TimesheetApplicationTests.java

```diff
@@ -57,6 +57,17 @@ public void roundTripWithDate() {
                 .contains(summaryFor(addedWorkingDay));
     }
 
+    @Test
```

Comment:
Could you add another test with a working day that has breaks, one break already exists but the time has changed and where another break was added. I'm curious if it will also save/update referenced (breaks) data.

Reply:
Added two tests for manipulating an existing break and adding a new one.